src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,10 @@
+// FOR STARS
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
+// FOR TRANSITION
+@mixin transition($smooth-color-transition) {
+  transition: $smooth-color-transition;
+}
```

Comment:
That's strange, if you you just passing full transition property to this mixin, you can just use variable